# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/Event.h
# Commit: f1e7580708c9
# Full Hash: f1e7580708c941ff3f6ee395462864f1645ef74e
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2025-01-11 09:16:48
# Regressor Bug: 1926530
# File Overlap Count: 1
# Description:
#   Bug 1926530 - skip certainly alive Events, r=mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D232945
# ==============================================================================

diff -r 18c7bd8144dd -r f1e7580708c9 dom/events/Event.h
--- a/dom/events/Event.h	Fri Jan 10 18:49:56 2025 +0000
+++ b/dom/events/Event.h	Fri Jan 10 18:58:51 2025 +0000
@@ -64,12 +64,8 @@
 #undef GENERATED_EVENT
 
 // IID for Event
-#define NS_EVENT_IID                                 \
-  {                                                  \
-    0x71139716, 0x4d91, 0x4dee, {                    \
-      0xba, 0xf9, 0xe3, 0x3b, 0x80, 0xc1, 0x61, 0x61 \
-    }                                                \
-  }
+#define NS_EVENT_IID \
+  {0x71139716, 0x4d91, 0x4dee, {0xba, 0xf9, 0xe3, 0x3b, 0x80, 0xc1, 0x61, 0x61}}
 
 class Event : public nsISupports, public nsWrapperCache {
  public:
@@ -91,7 +87,7 @@
 
  public:
   NS_DECL_CYCLE_COLLECTING_ISUPPORTS
-  NS_DECL_CYCLE_COLLECTION_WRAPPERCACHE_CLASS(Event)
+  NS_DECL_CYCLE_COLLECTION_SKIPPABLE_WRAPPERCACHE_CLASS(Event)
 
   nsIGlobalObject* GetParentObject() const { return mOwner; }
 
